backend/app/db/crud.py

```diff
@@ -0,0 +1,190 @@
+from app.db.psql_models import Attendance
+from app.db.psql_models import Course
+from app.db.psql_models import Lecturer
+from app.db.psql_models import Lesson
+from app.db.psql_models import Student
+from app.db.psql_models import StudentClass
+from app.db.psql_models import User
+from sqlmodel import select
+from sqlmodel import Session
+
+
+# StudentClass
+def get_studentclass_by_id(db: Session, studentclass_id: int):
+    return db.get(StudentClass, studentclass_id)
+
+
+def create_studentclass(db: Session, studentclass: StudentClass):
+    db_studentclass = StudentClass(
+            id=studentclass.id,
+            name=studentclass.name,
+            number_of_students=studentclass.number_of_students,
+            courses=studentclass.courses,
+            lecturers=studentclass.lecturers
+    )
```

Comment:
Do unpacking pls
```suggestion
    db_studentclass = StudentClass(**studentclass)
```

Reply:
They are now unpacked in the newest commit :D